### ISSUE TITLE
[question] Upgrade postgres from 9.6 to 10

### ISSUE THREAD

**Issue by f3l1x** [sameersbn/docker-gitlab#1686]
Hi,

Is it safe to upgrade from `sameersbn/postgresql:9.6-2` to `sameersbn/postgresql:10`? Or even from `sameersbn/postgresql:9.5-4` to `sameersbn/postgresql:10`?

Thank you guys! 👍

**Comment by gcelet** [comment]
Hi,

I'm running Gitlab CE on Postgresql 10 on Docker using sameersbn/gitlab for two weeks now.
I used to run on Postgresql 9.6. 
I upgraded GitLab CE to the last version at the time being (11.0.2). 
Then i upgraded Postgresql from 9.6 to 10: it's worked perfectly.
But i experienced a problem during upgrade from 11.0.2 to 11.1.0. 
It seems that Rails 4 and Postgresql 10 doesn't work together during sql migration: it's the problem that occurred to me.

I restored a backup and I'm staying for the moment on 11.0.2.

See these issues:
https://gitlab.com/gitlab-org/omnibus-gitlab/issues/2865
https://gitlab.com/gitlab-org/gitlab-ce/issues/14286

There are a fix possible that i didn't try:
https://github.com/rails/rails/issues/28780#issuecomment-354868174

My two cents.
